src/styles/index.scss

```diff
@@ -2,2 +1,244 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white-accent: #fff;
+$card-title-color: #34568b;
+
+.catalog {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: $main-accent;
   margin: 0;
+
+  &__header {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0 50px;
+    height: 60px;
+    box-sizing: border-box;
+  }
+
+  &__nav-logo {
+    width: 40px;
+    height: 40px;
+    display: flex;
+    align-items: center;
+  }
+
+  &__nav {
+    display: flex;
+    align-items: center;
+  }
+
+  &__nav-links {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  &__nav-list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    line-height: 60px;
+  }
+
+  &__nav-item {
+    display: inline-block;
+    margin-left: 17px;
+    text-align: center;
+    height: 60px;
+    line-height: 60px;
+  }
+
+  &__nav-item:first-child {
+    margin-left: 0;
+  }
+
+  &__nav-link {
+    text-transform: uppercase;
+    text-decoration: none;
+    cursor: pointer;
+    display: flex;
+    color: black;
+  }
+
+  &__nav-link:hover {
+    color: $blue-accent;
+  }
+
+  &__nav-link--is-active {
+    color: $blue-accent;
+    position: relative;
+  }
+
+  &__nav-link--is-active::after {
+    content: '';
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    height: 4px;
+    background-color: $blue-accent;
+    border-radius: 2px;
+  }
+
+  &__product-card {
+    display: grid;
+    grid-template-columns: repeat(1, 200px);
+    grid-template-rows: repeat(1, 408px);
+    gap: 46px 48px;
+
+    grid-auto-flow: row;
+    place-content: center;
+
+    padding: 50px 40px;
```

Comment:
The padding values seem to be arbitrary. For maintainability and consistency, consider using a set of standard spacing units throughout your project. This makes the design more coherent and the code easier to understand and maintain.